addons/website_project/static/src/js/options.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module **/
+
+import options from "@web_editor/js/editor/snippets.options";
+
+options.registry.WebsiteFormEditor = options.registry.WebsiteFormEditor.extend({
+    /**
+     * @override
+     */
+    async _renderCustomXML(uiFragment) {
+        await this._super(uiFragment);
+        const active_model=this.$target[0].dataset.model_name
+        let element = this.$target[0].querySelector(".s_website_form_send");
+        if (active_model=='project.task') {
+            element.innerText = 'Submit Task';
+        }
```

Comment:
```suggestion
        await this._super(...arguments);
        const activeModel = this.$target[0].dataset.model_name;
        if (active_model === 'project.task') {
           this.$target[0].querySelector(".s_website_form_send").innerText = _t('Submit Task');
        }
```

code is not tested.thanks